lib/Components/Cart/cart.css

```diff
@@ -0,0 +1,202 @@
+@import "/lib/Components/core.css";
+@import "/lib/Components/utilities.css";
+@import "/lib/Components/Header/header.css";
+@import "/lib/Components/Footer/footer.css";
+
+:root{
+    --COLOR-RED-60: rgb(255,0,0,0.6);  
+}
+
+/* MAIN STARTS */
+.sm-main-cart{
+    grid-area: main;
+    min-width: 100%;
+    height: 100%;
+    padding: 2rem 5rem;
+    display: grid;
+    grid-template-columns: 1fr 30rem;
+}
+
+.sm-cart-products-list{
+    display: flex;
+    justify-content: flex-start;
+    gap:2rem;
+    flex-wrap: wrap;
+    height: fit-content;
+}
+
+.sm-cart-products-list .sm-card{
+    width:90%;
+}
+
+.sm-cart-products-list .sm-card .sm-card-img{
+    background: var(--COLOR-WHITE-ALWAYS);
+    padding: 3rem;
+}
+
+.sm-cart-products-list .sm-card .sm-card-content{
+    padding: 1rem;
+    
+}
```

Comment:
this can be done with 2 classes as well, Do not increase the specificity until you dont have any other option